pandas/io/formats/info.py

```diff
@@ -83,30 +92,33 @@ def _initialize_memory_usage(
 
 
 class BaseInfo(ABC):
-    """Base class for DataFrameInfo and SeriesInfo.
+    """
+    Base class for DataFrameInfo and SeriesInfo.
 
     Parameters
     ----------
-    data : FrameOrSeries
+    data : DataFrame or Series
         Either dataframe or series.
     memory_usage : bool or str, optional
         If "deep", introspect the data deeply by interrogating object dtypes
         for system-level memory consumption, and include it in the returned
         values.
     """
 
-    def __init__(
-        self,
-        data: FrameOrSeries,
-        memory_usage: Optional[Union[bool, str]] = None,
-    ):
-        self.data = data
-        self.memory_usage = _initialize_memory_usage(memory_usage)
+    data: FrameOrSeriesUnion
```

Comment:
are we using FrameOrSeries? cc @simonjayhawkins

Reply:
see https://github.com/pandas-dev/pandas/pull/37868#discussion_r525126264